sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -185,121 +185,11 @@ The following section provides several code snippets illustrating common pattern
 ### Recognize Content
 Recognize text, tables, and selection marks like radio buttons and check boxes data, along with their bounding box coordinates, from documents.
 
-```C# Snippet:FormRecognizerSampleRecognizeContentFromUri
-Uri formUri = <formUri>;
-
-Response<FormPageCollection> response = await client.StartRecognizeContentFromUriAsync(formUri).WaitForCompletionAsync();
-FormPageCollection formPages = response.Value;
-
-foreach (FormPage page in formPages)
-{
-    Console.WriteLine($"Form Page {page.PageNumber} has {page.Lines.Count} lines.");
-
-    for (int i = 0; i < page.Lines.Count; i++)
-    {
-        FormLine line = page.Lines[i];
-        Console.WriteLine($"  Line {i} has {line.Words.Count} {(line.Words.Count == 1 ? "word" : "words")}, and text: '{line.Text}'.");
-
-        if (line.Appearance != null)
-        {
-            // Check the style and style confidence to see if text is handwritten.
-            // Note that value '0.8' is used as an example.
-            if (line.Appearance.Style.Name == TextStyleName.Handwriting && line.Appearance.Style.Confidence > 0.8)
-            {
-                Console.WriteLine("The text is handwritten");
-            }
-        }
-
-        Console.WriteLine("    Its bounding box is:");
-        Console.WriteLine($"    Upper left => X: {line.BoundingBox[0].X}, Y= {line.BoundingBox[0].Y}");
-        Console.WriteLine($"    Upper right => X: {line.BoundingBox[1].X}, Y= {line.BoundingBox[1].Y}");
-        Console.WriteLine($"    Lower right => X: {line.BoundingBox[2].X}, Y= {line.BoundingBox[2].Y}");
-        Console.WriteLine($"    Lower left => X: {line.BoundingBox[3].X}, Y= {line.BoundingBox[3].Y}");
-    }
-
-    for (int i = 0; i < page.Tables.Count; i++)
-    {
-        FormTable table = page.Tables[i];
-        Console.WriteLine($"  Table {i} has {table.RowCount} rows and {table.ColumnCount} columns.");
-        foreach (FormTableCell cell in table.Cells)
-        {
-            Console.WriteLine($"    Cell ({cell.RowIndex}, {cell.ColumnIndex}) contains text: '{cell.Text}'.");
-        }
-    }
-
-    for (int i = 0; i < page.SelectionMarks.Count; i++)
-    {
-        FormSelectionMark selectionMark = page.SelectionMarks[i];
-        Console.WriteLine($"  Selection Mark {i} is {selectionMark.State}.");
-        Console.WriteLine("    Its bounding box is:");
-        Console.WriteLine($"      Upper left => X: {selectionMark.BoundingBox[0].X}, Y= {selectionMark.BoundingBox[0].Y}");
-        Console.WriteLine($"      Upper right => X: {selectionMark.BoundingBox[1].X}, Y= {selectionMark.BoundingBox[1].Y}");
-        Console.WriteLine($"      Lower right => X: {selectionMark.BoundingBox[2].X}, Y= {selectionMark.BoundingBox[2].Y}");
-        Console.WriteLine($"      Lower left => X: {selectionMark.BoundingBox[3].X}, Y= {selectionMark.BoundingBox[3].Y}");
-    }
-}
-```
 For more information and samples see [here][recognize_content].
 
 ### Recognize Custom Forms
```

Comment:
Analyze Custom Documents

Reply:
I am not making modifications to the Readme in this PR. Will do in a future PR